packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.spec.js

```diff
@@ -28,16 +29,4 @@ describe('DoneAndTalkButton', function () {
       expect(wrapper.html()).to.be.null
     })
   })
-
-  describe('when props.completed is false', function () {
```

Comment:
Has props.completed been removed or are we just ignoring the false case in the tests now?

Reply:
In PFE, completed is a boolean being set by task completion validation functions per task type. None of that code has moved over and I'm not sure if we want to reuse any of it.